app/src/main/java/com/example/byitscover/ReviewPage.java

```diff
@@ -281,6 +282,7 @@ private void setBarnesAndNobleInfo(View view, BookListing listing) {
         }
         catch (Exception e) {
             System.out.println(e.toString());
+            banResultPrice.setText("$" + listing.getPrice().toString());
```

Comment:
`banResultPrice.setText("Pricing Info Not Provided");`?

Reply:
Same as above